### PR TITLE
MGMT-3551 add name to FIO check container

### PR DESCRIPTION
**Pull request by rollandf**

### CODE REVIEW

**Comment by danielerez** [comment]
@eranco74

**Comment by eranco74** [comment]
lgtm

File: internal/host/fioperfcheckcmd.go [hunk 1]
```diff
@@ -68,7 +68,7 @@ func (c *fioPerfCheckCmd) GetArgs() ([]string, error) {
 	}
 
 	arguments := []string{
-		"run", "--privileged", "--net=host", "--rm", "--quiet",
+		"run", "--privileged", "--net=host", "--rm", "--quiet", "--name=assisted-installer",
```
Comment (ronniel1) [inline internal/host/fioperfcheckcmd.go:71]: @rollandf did you make sure that all the other commands also have `--rm`?

**Pull request merged**